website/content/docs/using-netlify-cms-with-gatsby.md

```diff
@@ -0,0 +1,120 @@
+# Using Netlify CMS with Gatsby
+
+This guide will help you get started using Netlify CMS and Gatsby.
+
+## Create a new Gatsby site
+
+Let's start by importing the default Gatsby Starter Blog.
+
+```sh
+npx gatsby new blog https://github.com/gatsbyjs/gatsby-starter-blog
+```
+
+## Get to know Gatsby
+
+In your favorite code editor, open up the code generated for your "Gatsby Starter Blog" site, and take a look at the `content` directory.
+
+```sh
+cd blog
+code .      # If you are using Visual Studio Code
+```
+
+You will see that there are multiple `markdown` files that represent blog posts. Open one `.md` file and you will see something like this:
+
+```yml
+---
+title: New Beginnings
+date: "2015-05-28T22:40:32.169Z"
+description: This is a custom description for SEO and Open Graph purposes, rather than the default generated excerpt. Simply add a description field to the frontmatter.
+---
+
+Far far away, behind the word mountains, far from the countries Vokalia and
+Consonantia, there live the blind texts.
+```
+
+We can see above that each blog post has a title, a date, a description and a body. Now, let's recreate this using Netlify CMS. 
+
+## Add Netlify CMS to your site
+
+First let's install some dependencies. We'll need `netlify-cms` and `gatsby-plugin-netlify-cms`.
```

Comment:
I'd clarify here that the command should be run within the `blog` directory. Or to specify in the "open your text editor" step that one should `cd` into the blog directory and run all future commands in there (regardless of how people actually open their favorite text editor).